Dockerfile

```diff
@@ -40,10 +40,9 @@ RUN gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys 409B6B1796C27
   && echo rvm_silence_path_mismatch_check_flag=1 >> /etc/rvmrc \
   && echo 'install: --no-document\nupdate: --no-document' >> "/etc/.gemrc"
 
-
 WORKDIR /app
 
-ADD requirements.txt ./
+COPY . ./
```

Comment:
ADD does extra stuff we don't need, might as well just use COPY to copy the files. In addition, not sure why we have two COPY steps when they overlap, this is more layers and work for docker.